menas/ui/components/schema/schemaDetail.controller.js

```diff
@@ -173,7 +173,16 @@ sap.ui.define([
       if (status === 500) {
         const sSchemaType = this.byId("schemaFormatSelect").getSelectedItem().getText();
 
-        MessageBox.error(`Failed to upload new schema. Ensure that the file is a valid ${sSchemaType} schema and try again.`)
+        const oRawResponse = JSON.parse(oParams.getParameter("responseRaw"));
```

Comment:
Maybe this needs to be wrapped in a `try/catch`. Pretty sure that it would toss an exception if the response is not JSON.

Reply:
The default 500 response from tomcat is html